django/db/models/base.py

```diff
@@ -643,6 +643,8 @@ def save(self, force_insert=False, force_update=False, using=None,
                 # constraints aren't supported by the database, there's the
                 # unavoidable risk of data corruption.
                 if obj and obj.pk is None:
+                    if not field.related.multiple:
```

Comment:
As noted on the build failure, "Usage of field.related has been deprecated. Use field.remote_field instead."